<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhmAppSgbd</class>
 <widget class="QMainWindow" name="CIhmAppSgbd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1352</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>appSgbd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="lvTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>551</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbExtraire">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>541</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Extraire les utilisateurs</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbAjouter">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>651</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Ajouter</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>348</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="leNom"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lePnom"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leLogin"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leMdp">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pbAjouter">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>391</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter un utilisateur</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>250</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nom : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pr√©nom : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Login : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Mot de passe : </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbModifier">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>1021</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Modifier</string>
    </property>
    <widget class="QLabel" name="lId">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>140</y>
       <width>161</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>160</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Nom : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Login :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>348</width>
       <height>103</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="leModifNom"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leModifLogin"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>80</y>
       <width>411</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbEffacer">
        <property name="text">
         <string>Effacer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbModifier">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1352</width>
     <height>44</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
